src/recordlinker/database/mpi_service.py

```diff
@@ -190,14 +190,19 @@ def insert_blocking_values(
         session.commit()
 
 
-def get_patient_by_reference_id(
-    session: orm.Session, reference_id: uuid.UUID
-) -> models.Patient | None:
+def get_patients_by_reference_ids(
+    session: orm.Session, *reference_ids: uuid.UUID
+) -> list[models.Patient | None]:
```

Comment:
Nothing wrong in the code here, just curiosity: what's the impetus for changing this from a singular patient operating function to a plural patients? Just thinking about it pedagogically in the abstract, a single patient makes sense to me because we'll be operating on single incoming records, but I think this is an area where missing context is probably in play.

Reply:
The new endpoints referenced in the issue are accepting a list of patient ids.  This is an optimization as we can just get them all at once rather than querying the database N times for each patient id in the list.

https://github.com/CDCgov/RecordLinker/issues/158